### ISSUE TITLE
Framing with @explicit and @omitDefault

### ISSUE THREAD

**Issue by serwan** [lanthaler/JsonLD#65]
Hi,
I saw a strange behavior (and different from "offical playground") with theses two flags.
For @explicit  : if my frame specify a property which is missing from my input -> my output is empty.
On the playground, my output has the property I specified with "" as value

@omitDefault : isn't implemanted
On the playground, my output hasn't the property I specified in my frame if she's missing from my input

**Comment by bdecarne** [comment]
+1 for @omitDefault

**Comment by lanthaler** [comment]
Yeah, framing was never completed so the various implementations differ. Mine allows you to "filter" data whereas the Digital Bazaar's implementation does things slightly differently. What behavior would you expect? Why so?
